### DISCUSSION TITLE
posts/homelab-services-tour-2024/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#289] [Comments]
# posts/homelab-services-tour-2024/

What a year of self-hosting! Join me as we walk though my entire infrastructure and services that I have running in my HomeLab! This time I also include network diagrams and dive deep into which services I have running, where they are running, and why I chose them!

https://technotim.live/posts/homelab-services-tour-2024/

**Comment by CAL757** [comment]
Love your videos, they have helped me tons. I am starting to consider having a vlan with untrusted servers. To connect those to Rancher do you punch through your firewall or do you go around the firewall through the WAN?

I would love to see a video about having and securing Kubernetes in an untrusted environment. 

Also, why are you moving away from having one node in the untrusted vlans? I would like to see how you set up the nodes on other LANs and serve up reverse proxies for those untrusted nodes vs the trusted nodes.

**Comment by TSu-Energy** [comment]
Excellent Work
